docs/ref/contrib/gis/measure.txt

```diff
@@ -93,6 +93,7 @@ Unit Attribute                     Full name or alias(es)
 ``clarke_link``                    Clarke's link
 ``fathom``                         Fathom
 ``rod``                            Rod
+``furlong``                        Furlong
```

Comment:
`Furlong` -> `Furlong, Furrow long`

Reply:
Done. I'm going to leave the `Furlong` below the `Rod` in this documentation, as rods add up to furlongs (historically speaking) and the rest of this documentation isn't alphabetized. I can change that if you'd like.